docs/libraries.md

```diff
@@ -40,14 +37,14 @@ for ev3dev is written using this library.
 [GObjectIntrospection]: https://wiki.gnome.org/Projects/GObjectIntrospection
 [brickman]: https://github.com/ev3dev/brickman
 
-
 ## Extra languages
 We also have many great contrubutors that are maintaining extra libraries for
 languages not included in our other repository.  Note that some of these
-libraries may be outdated due to the fast development cycle of ev3dev.
+libraries may be outdated due to the fast development pace of ev3dev.
 
 * Extra languages:
     * [Go](https://github.com/ldmberman/GoEV3) updated for ev3dev-jessie by @ldmberman, [original](https://github.com/mattrajca/GoEV3) by @mattrajca
+    * [ev3dev-lang-java](https://github.com/ev3dev-lang-java/ev3dev-lang-java) an interface similar to that of LeJOS by @jabrena
```

Comment:
why put this between two Go libraries? wouldn't it be better to keep them together?